index.html

```diff
@@ -175,13 +175,83 @@ <h1>Skills</h1>
                     <li><span>&bull; GitLab</span></li>
                     <li><span>&bull; Terminal</span></li>
                 </ul>
-                <img class="about-img" src="assert/ic_languages_mob.svg" alt="lightbulb icon" />
+                <img class="about-img" src="/assert/ic_languages_mob.svg" alt="lightbulb icon" />
             </div>
+        </div>
+    </section>
+
+    <section class="form-section">
+        <div>
+            <a href="/index.html"> <img src="/assert/image_geometry_1.png" alt=""></a>
+        </div>
+
+
+        <div class="form-section1">
+            <div class="section">
+                <h1 class="section-h"> Get started</h1>
+                <p class="section-p"> Lorem ipsum dolor sit amet, consectetur
+                    adipiscing elit, sed do eiusmod tempor incididunt ut
+                    labore et dolore magna.</p>
+            </div>
+
+
+            <form action="https://formspree.io/f/xknyprke" method="POST">
+
+                <div class="form-container">
+                    <label for="name"></label>
+                    <input type="text" id="name" name="name" placeholder="name" required />
+                </div>
 
+                <div class="form-container">
+                    <label for="email"></label>
+                    <input type="email" id="email" name="e-mail" placeholder="Your email" required />
+                </div>
 
-            <div></div>
+                <div class="form-container">
+                    <label for="message">
+                        <textarea name="message" id="message" cols="30" rows="10" maxlength="500"
+                            placeholder="Enter your message here..." required></textarea>
+                    </label>
+                </div>
+
+                <button type="submit" class="btn form-btn ">
+                    Start collaboration
+                </button>
+
+            </form>
         </div>
     </section>
+
+    <footer class="personal-contact">
+        <div class="resume-con">
+            <p class="link-icon">Get my Resume</p>
+            <a href="/index.html"> <img class="link-image" src="./assert/ic_link_teal.png"
+                    alt="An simbol that work as link"></a>
+        </div>
```

Comment:
- [ ] As suggested by the previous reviewer, Kindly fix the background image at the top left as shown in the below video.
- [ ] Optional: It is highly recommended to implement this, Kindly make sure to align all the input fields with the same width as that of the Figma design.
- [ ]  As suggested by the previous reviewer kindly make sure that the placeholder text for the name is the same as in Figma design as "Your name".
- [ ] Kindly make sure that the button (Start Collaboration) is visible without hovering.
- [ ] Optional: It is highly recommended to implement this, Kindly add all the social media Icons as given in the Figma design.

https://user-images.githubusercontent.com/70269507/165522882-3f6f4059-855d-4fed-908f-798bde5d8c9b.mov